.jenkins/Jenkinsfile

```diff
@@ -16,7 +16,25 @@ def ACCTest(String label, String compiler, String unit, String suite) {
             checkout scm
 
             timeout(15) {
-                sh "./scripts/test-build-config -p $unit -b $suite --compiler=$compiler --disable_sim"
+                dir('build') {
+                    sh """
+                    if [ "${compiler}" == "gcc" ]; then
+                        export CC="gcc"
```

Comment:
Isn't there a way to set environment variables for a stage using Jenkins syntax? If possible, I'd prefer that to embedding more shell scripts.

Reply:
We could use `withEnv` like this:
`withEnv(["CC=gcc","CXX=g++"]) {
    sh ...
}`

Would that be ok?

Reply:
Yes! That's what I was looking for. Thank you 😄